substrate/primitives/transaction-pool/src/runtime_api.rs

```diff
@@ -46,10 +46,18 @@ sp_api::decl_runtime_apis! {
 		///
 		/// Note that this call may be performed by the pool multiple times and transactions
 		/// might be verified in any possible order.
+		#[changed_in(4)]
```

Comment:
This seems wrong, it should be this instead:
```suggestion
		#[changed_in(3)]
```